config.py

```diff
@@ -1,6 +1,23 @@
 import logging
 from app.vcap_services import get_database_uri
+from app import app
+from flasgger import Swagger
 
 SQLALCHEMY_DATABASE_URI = get_database_uri()
 SQLALCHEMY_TRACK_MODIFICATIONS = False
 LOGGING_LEVEL = logging.INFO
+
+# Configurue Swagger
+app.config['SWAGGER'] = {
```

Comment:
Use syntax similar to line 6, 7, 8, e.g. 
SWAGGER = {...}
instead of 
app.config['SWAGGER'] = {...}

Reply:
addressed